src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -70,6 +70,7 @@ def load_arguments(self, _):
         c.argument('name', options_list=['--name', '-n'], help="Name of the new app service plan", completer=None)
         c.argument('sku', arg_type=sku_arg_type)
         c.argument('is_linux', action='store_true', required=False, help='host webapp on Linux worker')
+        c.argument('hyper_v', action='store_true', required=False, help='host webapp on Windows container')
```

Comment:
i suggest the name be "--use-hyper-v" instead of a noun

Reply:
Hi @yugangw-msft - We are also checking in the equivalent powerShell cmdlets and are using HyperV over there. Can we keep the current parameter name for consistency?